mail_template_conditional_attachment/models/mail_template.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Druidoo - Iván Todorovich <ivan.todorovich@druidoo.io>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models, fields
+from odoo.tools import pycompat
+
+
+class MailTemplate(models.Model):
+    _inherit = 'mail.template'
+
+    conditional_attachment_ids = fields.One2many(
+        'mail.template.conditional.attachment',
+        'mail_template_id',
+        string='Conditional Attachments',
+    )
+
+    @api.multi
+    def generate_email(self, res_ids, **kwargs):
+        """ Overload this method to attach conditional attachments """
```

Comment:
Avoid overriding the docstring because you wipe the original. Use a comment instead.

Reply:
Got it :+1: